pykafka/rdkafka/_rd_kafkamodule.c

```diff
@@ -471,7 +471,7 @@ RdkHandle_start(RdkHandle *self,
     /* Set logger and brokers */
     int brokers_added;
     Py_BEGIN_ALLOW_THREADS  /* avoid callbacks deadlocking */
-        rd_kafka_set_logger(self->rdk_handle, logging_callback);
+        rd_kafka_conf_set_log_cb(self->rdk_conf, logging_callback);
```

Comment:
Thanks for picking up on this. It needs a little reshuffling, because `rdk_conf` gets clobbered by `rd_kafka_new()` (which is not obvious without the librdkafka api docs). I'll go figure it out.